templates/shopapi/home.html

```diff
@@ -0,0 +1,18 @@
+{% extends './base.html' %}
+
+
+{% block content %}
+    {% if user.is_authenticated %}
+    <div style="margin-left: 3%">
```

Comment:
@vasyltsar Duplication.  You can use this block outside "if else" statements `<div style="margin-left: 3%">`